### PR TITLE
SDK - Exchange Guide - Add Python and Rust Code 

### PR DESCRIPTION
**Pull request by lucas-tortora**
# Description of change

* added rust and python code to the exchange guide
* added api refs to the exchange guide
* edited exchange-guide
* fixed backup and security title

## Links to any relevant issues

fixes https://github.com/iotaledger/devx/issues/171

## Type of change

- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx [hunk 1]
```diff
@@ -12,27 +12,12 @@ keywords:
...
-](https://wiki.iota.org/shimmer/develop/explanations/what-is-stardust/) for an overview of its features or [TIPs](https://github.com/iotaledger/tips) flagged `Stardust` for a deeper understanding of the protocol changes.
-
-:::
-
-:::warning
-
-The Stardust update allows for complex tokenization schemes that can lead to monetary losses if misused.
-Transaction outputs might have multiple unlocking conditions. These conditions may require the return of part or all of the output amounts, have an expiration date if not claimed on time, or they may not be unlockable for an extended period of time. Therefore, you need to review these unlock conditions carefully before accepting any deposits to avoid loss of funds.
-
-:::
```
Comment (Dr-Electron) [inline shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:32]: Is there a reason why this got removed?

File: shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx [hunk 2]
```diff
@@ -43,63 +28,175 @@ Features of the IOTA SDK:
...
+
+```sh
+npm i @iota/sdk dotenv
+```
+
+</TabItem>
+<TabItem value="yarn" label="Yarn">
+
+```sh
+yarn add @iota/sdk
```
Comment (Dr-Electron) [inline shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:173]: We could use npm2yarn instead 🤔 Wdyt?  https://docusaurus.io/docs/2.3.1/markdown-features/code-blocks#npm2yarn-remark-plugin

File: shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx [hunk 3]
```diff
@@ -43,63 +28,175 @@ Features of the IOTA SDK:
...
+
+</TabItem>
+<TabItem value="python" label="Python">
+
+Check out our [Getting Started with Python](../getting-started/python.mdx) guide for more detailed instructions.
+
+```sh
+pip install iota-sdk
+pip install -r requirements-dev.txt
+pip install .
```
Comment (Dr-Electron) [inline shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:187]: Why would this be needed? You already installed the SDK, no?

**Review by Dr-Electron — CHANGES_REQUESTED** [review]
Other than the stuff mentioned LGTM. Do we need a review from the SDK team?

File: shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx (hunk 1 above)
Comment (lucas-tortora) [reply shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:32]: cause we have 3 warnings for this in the doc.

File: shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx (hunk 3 above)
Comment (lucas-tortora) [reply shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:187]:
```suggestion
```

**Comment by lucas-tortora** [comment]
> Other than the stuff mentioned LGTM. Do we need a review from the SDK team?

File: shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx (hunk 1 above)
Comment (Dr-Electron) [reply shimmer/docs/iota-sdk/docs/how-tos/exchange-guide.mdx:32]: Realized that too after i finished reading 😅

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**